install/installer/traf_config_setup

```diff
@@ -226,6 +230,33 @@ echo "export NODE_LIST=\"$NODE_LIST\"" >> $LOCAL_TRAF_CONFIG
 node_count=$(echo $NODE_LIST | wc -w)
 
 #==============================================
+echo -n "Are management nodes enabled (Y/N), default is N: "
+read answer
+ 
+if [ -z $answer ]; then
+      echo "export MANAGE_ENABLED=\"N\"" >> $LOCAL_TRAF_CONFIG
+else
+      if [[ "${answer}" =~ ^[Yy]$ ]]; then
+         echo "export MANAGE_ENABLED=\"Y\"" >> $LOCAL_TRAF_CONFIG
+         #List of Management Nodes
+         echo -n "Enter list management nodes (blank separated), default [$MANAGE_NODES]: "
+         read answer
+         if [[ -z "$answer" ]]; then
+            if [ -z "$MANAGE_NODES" ]; then
+               echo "***ERROR: Must enter list of management nodes."
+               exit -1
+            fi
+         else
+            MANAGE_NODES="$answer"
```

Comment:
Should be MANAGEMENT_NODES instead of MANAGE_NODES. Should fix all files where this is referenced